bin/logdaemon.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
`python2` isn't there everywhere, so we need to be more careful with this somehow, maybe...

I don't have `python2` on OS X, for example...